examples/tac_deploy/tac-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tac-deploy
+spec:
+  selector:
+    matchLabels:
+      app: tac-deploy
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tac-deploy
+      namespace: aea-research
+    spec:
+      nodeSelector:
+        # type: agent-test
+        kubernetes.io/os: linux
+      containers:
+      - name: tac-deploy-container
+        image: gcr.io/fetch-ai-sandbox/tac_deploy:0.0.1
+        env:
+         - name: PARTICIPANTS_AMOUNT
+           value: "70"
+         - name: MINUTES_TILL_START
+           value: "10"
+         - name: COMPETITION_TIMEOUT
+           value: "86400"
+         - name: INACTIVITY_TIMEOUT
+           value: "3600"
+         - name: LOG_LEVEL
+           value: "INFO"
```

Comment:
In recent deployment, the logs are still at DEBUG level, so not sure what's propagated wrongly where